### ISSUE TITLE
[Bug]: Color control automatically pops up, without triggering the control itself

### ISSUE THREAD

**Issue by valentinpalkovic** [storybookjs/storybook#28511]
### Describe the bug

When visiting a story with a color control, the control will automatically open

### Reproduction steps

1. Go to https://634ff0d0ec053b270775979d-vpmrbjlyny.chromatic.com/?path=/story/addons-controls-basics--undefined
2. In the controls panel, the color picker is automatically opened

![image](https://github.com/storybookjs/storybook/assets/5889929/88c0ce72-abb6-42f8-823d-71a2945c1b7f)

**Comment by JohnMikko27** [comment]
Hi does this issue still need help? Someone tried to make a PR 2 over 2 months ago but didn't successfully merge. Can I start working on this issue?

**Comment by janeedgina** [comment]
Hi! I'm new to contributing and I would like to work on this issue. Can I solve this?

**Comment by shilman** [comment]
This is not a bug. This is just how the story is configured. @valentinpalkovic @vanessayuenn We can just update the story.

**Comment by shilman** [comment]
@ghengeveld  did you have a use case in mind for why we'd want to be able to have the color control open by default?

**Comment by ghengeveld** [comment]
@shilman No, I think this was done just for demonstration and convenience, to make it obvious the story is related to the color control.
